Gemfile

```diff
@@ -72,3 +72,12 @@ group :test do
 end
 
```

Comment:
![image](https://github.com/ilyassatlassi/Blog-app/assets/93039359/198ff2f0-91dd-4a2b-a1ef-dd09bd2c18e6)

- [ ] It would be great if you fix your Github link in the Authors section of your readme file because it is broken :fearful: Kindly fix it to make your readme descriptive and professional :+1: